src/hooks/useIsWrongChain.ts

```diff
@@ -1,10 +1,11 @@
 import useChainId from '@/hooks/useChainId'
 import useWallet from '@/hooks/wallets/useWallet'
 
+// TODO: Remove this later
```

Comment:
👀

Reply:
I saw that this was now removed. We can also remove the `ChainSwitcher` as well.